src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,67 @@
+package mate.jdbc.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers = new ArrayList<>();
+
+    public Car(Long id, String model, Manufacturer manufacturer) {
+        this.id = id;
+        this.model = model;
+        this.manufacturer = manufacturer;
+    }
+
+    public Car(String model, Manufacturer manufacturer) {
+        this.model = model;
+        this.manufacturer = manufacturer;
+    }
+
+    public Car() {
```

Comment:
It's recommended to create a constructor with all fields except 'id' to avoid creating an empty instance of the Car class. Consider removing the empty constructor.